internal_workflow_testsuite.go

```diff
@@ -257,19 +258,34 @@ func (env *testWorkflowEnvironmentImpl) newTestWorkflowEnvironmentForChild(optio
 	return childEnv
 }
 
-func (env *testWorkflowEnvironmentImpl) setActivityTaskList(tasklist string, activityFns ...interface{}) {
+func (env *testWorkflowEnvironmentImpl) setActivityTaskList(
+	tasklist string,
+	ao RegisterActivityOptions,
+	activityFns ...interface{},
+) {
 	for _, activityFn := range activityFns {
-		env.testSuite.RegisterActivity(activityFn)
+		env.getHostEnv().RegisterActivityWithOptions(activityFn, ao)
```

Comment:
we want test to use both the private host env and also the global host env as well, so that activity/workflow registered by init() method should be available to tests and don't require tests to register them via the private host again.